src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx

```diff
@@ -1007,6 +1007,52 @@ This section defines the Node.js agent variables in the order they typically app
 
 This section allows you to choose which API methods are enabled. Each configuration option allows you to modularly enable API methods that are responsible for sending custom information to New Relic.
 
+<CollapserGroup>
+  <Collapser
+    id="custom-attributes"
+    title="esm.custom_instrumentation_entrypoint"
+  >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            String
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            (none)
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_API_ESM_CUSTOM_INSTRUMENTATION_ENTRYPOINT`
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    Relative filepath to custom instrumentation definition, which is relative to the application root. Enables the ability to load custom instrumentation in ES module applications.
```

Comment:
@jmartin4563, how about a minor revision like this: 

This uses the relative filepath to the custom instrumentation definition, which is relative to the application root. It enables you to load custom instrumentation in ES module applications.